sklearn/feature_extraction/image.py

```diff
@@ -331,33 +331,38 @@ def extract_patches_2d(image, patch_size, max_patches=None, random_state=None):
     Returns
     -------
     patches : array, shape = (n_patches, patch_height, patch_width) or
-         (n_patches, patch_height, patch_width, n_channels)
-         The collection of patches extracted from the image, where `n_patches`
-         is either `max_patches` or the total number of patches that can be
-         extracted.
+        (n_patches, patch_height, patch_width, n_channels)
+        The collection of patches extracted from the image, where `n_patches`
+        is either `max_patches` or the total number of patches that can be
+        extracted.
 
     Examples
     --------
-    >>> import numpy as np
+    >>> from sklearn.datasets import load_sample_images
     >>> from sklearn.feature_extraction import image
-    >>> one_image = np.arange(16).reshape((4, 4))
-    >>> one_image
-    array([[ 0,  1,  2,  3],
-           [ 4,  5,  6,  7],
-           [ 8,  9, 10, 11],
-           [12, 13, 14, 15]])
+    >>>
+    >>> # Use the array data from the first image in this dataset:
+    >>> one_image = load_sample_images().images[0]
+    >>> print('Image shape: {}'.format(one_image.shape))
+    Image shape: (427, 640, 3)
+    >>>
     >>> patches = image.extract_patches_2d(one_image, (2, 2))
-    >>> patches.shape
-    (9, 2, 2)
-    >>> patches[0]
-    array([[0, 1],
-           [4, 5]])
-    >>> patches[1]
-    array([[1, 2],
-           [5, 6]])
-    >>> patches[8]
-    array([[10, 11],
-           [14, 15]])
+    >>> print('Patches shape: {}'.format(patches.shape))
+    Patches shape: (272214, 2, 2, 3)
+    >>>
+    >>> # Here are just two of these patches:
+    >>> print(patches[1]) # doctest: +NORMALIZE_WHITESPACE \
+                                     +DONT_ACCEPT_BLANKLINE
```

Comment:
I don't think you should need `DONT_ACCEPT_BLANKLINE`. Are you sure it's necessary?

Reply:
@CatChenal can you give it a try without the `DONT_ACCEPT_BLANKLINE` option?